### DISCUSSION TITLE
There is no way to send a SIGINT within Windows when running DevProxy in the background

### DISCUSSION THREAD

**Discussion by RobertHarwig** [dotnet/dev-proxy#1220] [Q&A] [answered]
How can this be done in Windows?

**Comment by waldekmastykarz** [comment, marked as answer]
In the latest version of Dev Proxy we introduced web API, which you can use to gracefully stop Dev Proxy. For more information see: https://learn.microsoft.com/en-us/microsoft-cloud/dev/dev-proxy/technical-reference/proxy-api#post-proxystopproxy

**Comment by waldekmastykarz** [comment]
Closing as completed for the lack of further questions.
